tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -68,7 +68,7 @@
                 ),
                 category3.ifThenAlso(
                     condition_field_name="SANC_REDUCTION_AMT",
-                    condition_function=category3.isGreaterThan(0),
+                    condition_function=category3.isGreaterThan(0, inclusive=True),
```

Comment:
@klinkoberstar @elipe17 -- since Item 26AIII (Family Sanction) is deprecated, and already allows for 0, i think this cat 3 validator can be dropped. i think if it were to change to accommodate the deprecated field, i believe we'd want to relax line 73 instead of line 71.

Reply:
Just removed the validator and pushed the change